examples/simple_example/main.tf

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+provider "google" {
+  version = "~> 1.20"
+  project = "${var.project_id}"
+  region  = "${var.region}"
+  zone    = "${var.region}-a"
+}
+
+module "startup-scripts" {
+  source = "../../"
+}
```

Comment:
@adrienthebo Regarding your project_id and region comment in https://github.com/terraform-google-modules/terraform-google-startup-scripts/pull/1#issuecomment-448393478 I verified the steps as documented and fixed up in 0631423..9458a1d . Please let me know if the simple use case of `module "startup-scripts" { source = "../../" }` does not work for you.